### PR TITLE
[BD-46] feat: Style dictionary setting

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
## Description

- create all necessary JSON files according to the schema defined in "Categorize Paragon's";
- configure style-dictionary to include the following platform: CSS variables, SCSS variables.
- nice to have CSS utility classes (may need a custom formatter)

## Merge Checklist

* [x] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [x] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [x] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [x] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [x] Were your changes tested in the `example` app?
* [ ] Is there adequate test coverage for your changes?
* [x] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please add `wittjeff` and `adamstankiewicz` as reviewers on this PR.

## Post-merge Checklist

* [x] Verify your changes were released to [NPM](https://www.npmjs.com/package/@edx/paragon) at the expected version.
* [x] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [x] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!


When this pull request is ready, tag your edX technical lead.

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "commit": "commit",
     "debug-test": "node --inspect-brk node_modules/.bin/jest --runInBand --coverage",
     "stylelint": "stylelint \"src/**/*.scss\" \"scss/**/*.scss\" \"www/src/**/*.scss\" --config .stylelintrc.json",
-    "lint": "npm run stylelint && eslint --ext .js --ext .jsx --ext .ts --ext .tsx . && npm run generate-component-lint",
+    "lint": "npm run stylelint && eslint --ext .js --ext .jsx . && npm run generate-component-lint",
```
Comment (monteri) [inline package.json:33]: `--ext .ts --ext .tsx .`

File: tokens/z-index/base.tokens.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "zindex": {
+    "dropdown": { "value": "1000", "var": "zindex-dropdown" },
```
Comment (monteri) [inline tokens/z-index/base.tokens.json:3]: original

File: package.json [hunk 3]
```diff
@@ -56,8 +57,10 @@
     "file-selector": "^0.6.0",
     "font-awesome": "^4.7.0",
     "glob": "^8.0.3",
+    "i": "^0.3.7",
     "lodash.uniqby": "^4.7.0",
     "mailto-link": "^2.0.0",
+    "npm": "^8.12.2",
```
Comment (monteri) [inline package.json:63]: `i` and `npm` I should not be here

File: tokens/elevation-shadows/box-shadow/box-shadow.tokens.json [hunk 4]
```diff
@@ -0,0 +1,114 @@
+{
+  "box-shadow": {
+    "down": {
+      "1": {
+        "value": "0 .0625rem .125rem rgba(0, 0, 0, .15), 0 .0625rem .25rem rgba(0, 0, 0, .15)",
```
Comment (monteri) [inline tokens/elevation-shadows/box-shadow/box-shadow.tokens.json:5]: Probably we need `"themable": true` here too

Comment (PKulkoRaccoonGang) [reply tokens/elevation-shadows/box-shadow/box-shadow.tokens.json:5]: done

File: package.json (hunk 3 above)
Comment (PKulkoRaccoonGang) [reply package.json:63]: done

**Comment by openedx-webhooks** [comment]
@PKulkoRaccoonGang 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/1571&entry.1484655318=https://github.com/PKulkoRaccoonGang&entry.752974735=2022-08-18+21:47&entry.1917517419=2022-09-02+08:20&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**